### PR TITLE
Fix/centralize display settings

### PR DESCRIPTION
**Pull request by santiagonoguez**
This is the final fix, with the changes discussed this morning during Standup. I've also tested it manually.

### CODE REVIEW

File: installer/main-controller.js [hunk 1]
```diff
@@ -163,11 +163,17 @@ function ready() {
   });
 }
 
+function logInstallComplete() {
+  const displayId = config.getInstallOrUpgradeDisplayId();
+
+  log.all("install complete", displayId, 100);
```
Comment (andrecardoso) [inline installer/main-controller.js]: Is this going to log the temporary display id on new installs?

Comment (santiagonoguez) [reply installer/main-controller.js]: @andrecardoso yes, that's why the function was called getInstallOrUpgradeDisplayId, and not just getDisplayId

Comment (andrecardoso) [reply installer/main-controller.js]: I think in this case we want to log the display id in the event_details as we're already logging the temporary display id in the display_id field

Comment (santiagonoguez) [reply installer/main-controller.js]: @andrecardoso No, with this PR the display_id field no longer contains the temporary display id, but the definitive one. That's why I'm adding the temporary display id on event_details for new installs and the reliability queries will need to read it from here: https://github.com/Rise-Vision/bigquery-queries/pull/59

Comment (andrecardoso) [reply installer/main-controller.js]: @santiagonoguez sounds good. Thanks

**Comment by santiagonoguez** [comment]
@andrecardoso @rudibravo switched to option A, also tested it manually, please review again

**Pull request merged**